streamprocessor/src/main/java/org/md2k/streamprocessor/output/PuffProbability.java

```diff
@@ -0,0 +1,69 @@
+package org.md2k.streamprocessor.output;
+
+import android.content.Context;
+
+import org.md2k.datakitapi.datatype.DataTypeDouble;
+import org.md2k.datakitapi.source.METADATA;
+import org.md2k.datakitapi.source.datasource.DataSourceBuilder;
+import org.md2k.datakitapi.source.datasource.DataSourceType;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/*
+ * Copyright (c) 2016, The University of Memphis, MD2K Center
+ * - Syed Monowar Hossain <monowar.hossain@gmail.com>
```

Comment:
You can claim full credit for this file unless Monowar has helped.  Either way, you should put your name first